val-ansible-role/valkey/tasks/standalone.yml

```diff
@@ -0,0 +1,25 @@
+- name: Install Valkey
```

Comment:
task name should be like
**standalone | Install Valkey**
follow below for details
https://ansible.readthedocs.io/projects/lint/rules/name/#nameprefix